app/__init__.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
```

Comment:
Remove commented code if at all you do not need them for the apps functionality.